sbi/simulators/simutils.py

```diff
@@ -88,21 +97,34 @@ def simulate_in_batches(
         logging.warning("Zero-length parameter theta implies zero simulations.")
         x = torch.tensor([])
     elif sim_batch_size is not None and sim_batch_size < num_sims:
-        # Dev note: pyright complains of torch.split lacking a type stub
-        # as of PyTorch 1.4.0, see https://github.com/microsoft/pyright/issues/291
         batches = torch.split(theta, sim_batch_size, dim=0)
 
+        num_batches = len(batches)
```

Comment:
maybe call it `num_worker_batches` to make clear that this does not stem from `simlation_batch_size`

Reply:
I think the point is that is actually _is_ the number of simulation batches, dont you agree?

Reply:
But you only ues it once more, so perhaps don't give it a name.

Reply:
Hm, maybe I do not fully get it.

But my point would be that, during simulation, we have two kinds of batches:
1) the batches handled by each worker (size will be `theta[0]/pbar_steps`)
2) the `simultion_batch_size`

It would be good if the name of the `num_batches` variable indicated that it refers to the "first" kind of batch I outline above.

Hence my suggestion `num_worker_batches`